modules/catalog-compaction/src/main/java/org/apache/ignite/internal/catalog/compaction/CatalogCompactionRunner.java

```diff
@@ -376,10 +406,110 @@ private static List<String> missingNodes(Set<String> requiredNodes, Collection<L
         return requiredNodes.stream().filter(not(logicalNodeIds::contains)).collect(Collectors.toList());
     }
 
+    CompletableFuture<Void> propagateTimeToReplicas(long minimumRequiredTime) {
+        Map<Integer, Integer> tablesWithPartitions = catalogManagerFacade.collectTablesWithPartitionsBetween(
```

Comment:
Could you please elaborate what partition between (minimumRequiredTime, now()) means?

Reply:
Added javadoc for `collectTablesWithPartitionsBetween`